src/hooks/useProvidersList.js

```diff
@@ -271,23 +278,30 @@ const useProvidersList = (options, updatedAt = null) => {
   }, [filters]);
 
   const loadProviders = () => {
-    if (storeSession.tradeApi.accessToken) {
+    if (storeSession.tradeApi.accessToken && connectedProviders) {
```

Comment:
Why are we waiting for connectedProviders to be received before making the request? I think it can be optimized.